test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageControllerSpec.scala

```diff
@@ -1787,22 +1788,22 @@ class MultipleDisposalsTriageControllerSpec
             }
 
             checkIsRedirect(
-              performAction(key -> "2024"),
+              performAction(key -> s"$currentTaxYear"),
               routes.MultipleDisposalsTriageController.checkYourAnswers()
             )
           }
 
-          "user has not answered the tax year exchanged section and selects after April 06th, 2023" in {
+          "user has not answered the tax year exchanged section and selects after currentTaxYear minus 1" in {
             val taxYear = sample[TaxYear].copy(
-              startDateInclusive = LocalDate.of(2023, 4, 6),
-              endDateExclusive = LocalDate.of(2024, 4, 6)
+              startDateInclusive = LocalDate.of(currentTaxYear - 1, 4, 6),
+              endDateExclusive = LocalDate.of(currentTaxYear, 4, 6)
             )
 
             inSequence {
               mockAuthWithNoRetrievals()
               mockGetSession(session)
-              mockAvailableTaxYears()(Right(List(2023)))
-              mockGetTaxYear(todayTaxYear2023)(Right(Some(taxYear)))
+              mockAvailableTaxYears()(Right(List(currentTaxYear - 1)))
```

Comment:
Trivial. You can use `currentTaxYearMinus1` here than again deriving from `currentTaxYear`. Same for other cases like this